ska_ftp/tests/test_basic.py

```diff
@@ -10,11 +10,10 @@
 
 logger = pyyaks.logger.get_logger()
 
-NETRC = ska_ftp.parse_netrc()
 LUCKY = 'lucky.cfa.harvard.edu'
 
 
-def roundtrip(FtpClass, logger=None):
+def roundtrip(NETRC, FtpClass, logger=None):
```

Comment:
Don't use an all-caps here, especially since it masks a global value (fixture) making things super confusing.

Optionally also change `FtpClass` to `ftp_cls` to comply with modern naming standards.

Reply:
I agree about not using all-caps here.